### PR TITLE
[of_topology] Update README.rst and openapi.yml

### LINKED ISSUE #298: [of_topology] Update README.rst and openapi.yml

### PR DESCRIPTION
**Pull request by macartur**
Update listened and generated events
Update Rest endpoints
Fix #298

### CODE REVIEW

File: napps/kytos/of_topology/main.py [hunk 1]
```diff
@@ -77,9 +77,11 @@ def shutdown(self):
     def get_json_topology(self):
         """Return a json with topology details.
 
-        Method responsible to return a json in /kytos/topology route.
+        Method responsible to return a json in api/legacy/of_topology/topology
```
Comment (diraol) [inline napps/kytos/of_topology/main.py]: `s/responsible to return/responsible for returning/`

File: napps/kytos/of_topology/openapi.yml [hunk 2]
```diff
@@ -15,16 +16,16 @@ info:
...
     get:
       summary: Return a json with topology details.
       description: Sample description
       tags:
         - Current
       responses:
         200:
-          description: "Method responsible to return a json in /kytos/topology
-          route."
+          description: "Method responsible to return a json in
```
Comment (diraol) [inline napps/kytos/of_topology/openapi.yml]: `s/responsible to/responsible for/`

File: napps/kytos/of_topology/openapi.yml [hunk 3]
```diff
@@ -4,8 +4,9 @@ info:
   version: 0.1.1
   description: "**Warning**: *This documentation is experimental and will change soon.*\n
 
-  Keeps track of links between hosts and switches. Returns a JSON
-  with network topology using the endpoint /legacy/topology."
+  Keeps track of links between hosts and switches. Returns a JSON with network
+  topology using the endpoint /api/legacy/of_topology/topology.
+  "
```
Comment (cemsbr) [inline napps/kytos/of_topology/openapi.yml]: Keep track ... and return a JSON ...

Note: use imperative like in docstrings.

File: napps/kytos/of_topology/openapi.yml (hunk 2 above)
Comment (cemsbr) [reply napps/kytos/of_topology/openapi.yml]: Return a json (...)

File: napps/kytos/of_topology/README.rst [hunk 4]
```diff
@@ -34,6 +34,19 @@ If you are going to install kytos-napps from source code, all napps will be
...
 running).
 
+Events
+======
+
+Listen
+------
+kytos/of_core.v0x01.messages.in.ofpt_packet_in
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  Listen to any input of OpenFlow PacketIn in versions 1.0 (v0x01).
```
Comment (cemsbr) [inline napps/kytos/of_topology/README.rst]: Listen to any OpenFlow 1.0 PacketIn message.

File: napps/kytos/of_topology/README.rst [hunk 5]
```diff
@@ -34,6 +34,19 @@ If you are going to install kytos-napps from source code, all napps will be
...
+
+kytos/of_core.v0x01.messages.in.ofpt_port_status
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  Listen to any input of OpenFlow PortStatus in versions 1.0 (v0x01).
```
Comment (cemsbr) [inline napps/kytos/of_topology/README.rst]: Same as above

**Comment by cemsbr** [comment]
Issue moved to [kytos/of_topology #7](https://github.com/kytos/of_topology/issues/7) via [**ZenHub**](https://www.zenhub.com/)

**Pull request closed without merging**